src/main/kotlin/org/rust/cargo/toolchain/RustToolchain.kt

```diff
@@ -181,14 +158,14 @@ private fun parseRustcVersion(lines: List<String>): Version? {
 
     val major = releaseMatch.groups[1]!!.value.toInt()
```

Comment:
i wondered if we can have a safer way to get these numbers from regex match results instead use `!!` directly.